### PR TITLE
fixes issue/1331

### PR DESCRIPTION
**Pull request by lc-thomasberger**
add more screen related properties to Adapt.device object

### CODE REVIEW

File: src/core/js/device.js [hunk 1]
```diff
@@ -44,9 +44,22 @@ define(function(require) {
...
     }
 
+    function getScreenHeight() {
+        return isAppleDevice()
+            ? getAppleScreenHeight()
+            : window.innerHeight || $window.height();
+    }
+
+    function getScreenOrientation() {
+        return (Adapt.device.screenWidth >= Adapt.device.screenHeight) ? 'landscape' : 'portrait';
```
Comment (oliverfoster) [inline src/core/js/device.js]: I seem to remember the screen height doesn't change when the orientation changes on iOS. So you'd always have to use the window.innerWidth to calculate the orientation. I'll see if that applies here a bit later and find some references if applicable.

File: src/core/js/device.js [hunk 2]
```diff
@@ -77,6 +90,10 @@ define(function(require) {
         return (Math.abs(window.orientation) === 90) ? screen.height : screen.width;
     }
 
+    function getAppleScreenHeight() {
+        return (Math.abs(window.orientation) === 90) ? screen.width : screen.height;
```
Comment (oliverfoster) [inline src/core/js/device.js]: Cool. Ignore my last comment. :+1:

File: src/core/js/device.js [hunk 3]
```diff
@@ -4,10 +4,12 @@ define(function(require) {
...
-    Adapt.device = {
-        touch: Modernizr.touch,
-        screenWidth: getScreenWidth()
-    };
+    Adapt.device = {};
+    Adapt.device['touch'] = Modernizr.touch;
+    Adapt.device['screenWidth'] = getScreenWidth();
+    Adapt.device['screenHeight'] = getScreenHeight();
+    Adapt.device['orientation'] = getScreenOrientation();
+    Adapt.device['aspectRatio'] = getScreenAspectRatio();
```
Comment (brian-learningpool) [inline src/core/js/device.js]: Was there a reason for switching to bracket notation here?

Comment (lc-thomasberger) [inline src/core/js/device.js]: Yes, to be able to reuse the functions. When called from the object, Adapt.device is not defined.

Comment (oliverfoster) [inline src/core/js/device.js]: You can do ```Adapt.device.touch = ``` is what he means.

**Pull request merged**